esp/esp/survey/views.py

```diff
@@ -37,6 +37,7 @@
 
 import datetime
 import xlwt
+import re
```

Comment:
Fine with me.  Some of these may be bugfixes worth keeping, but we can always apply them later.